client/.gitignore

```diff
@@ -0,0 +1,25 @@
+# See https://help.github.com/articles/ignoring-files/ for more about ignoring files.
+
+# dependencies
+/node_modules
+/.pnp
+.pnp.js
+node_modules/
+.env
+
+# testing
+/coverage
+
+# production
```

Comment:
Not sure why it won't let me comment on the files, but since database.js and index.js are empty, we can delete them since Tyson and Jerry will add in a new file. Therefore, we can remove the server folder inside backend.